.github/workflow_scripts/lint_check.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+set -ex
+
+source $(dirname "$0")/env_setup.sh
+
+setup_lint_env
+
+python3 -m tox -e format
+python3 -m tox -e lint
+python3 -m tox -e isort
```

Comment:
I believe this won't work at all cause you didn't include the config file: https://github.com/autogluon/autogluon-cloud/blob/master/setup.cfg